### PR TITLE
feat: Add InlineCard component

### PR DESCRIPTION
**Pull request by GoOz**
Taken from Banks, here is `InlineCard` (previously known as `TextCard`)

Preview to come…

### CODE REVIEW

**Comment by GoOz** [comment]
[Faire le composant Inline Card dans UI (à rapatrier de TextCard de Banks)](https://trello.com/c/Bzo1CBNh/1549-faire-le-composant-inline-card-dans-ui-%C3%A0-rapatrier-de-textcard-de-banks)

**Comment by ptbrowne** [comment]
Awesome, thanks :tada:

File: react/__snapshots__/examples.spec.jsx.snap [hunk 1]
```diff
@@ -1271,6 +1271,8 @@ exports[`Infos should render examples: Infos 1`] = `
 </div>"
 `;
 
+exports[`InlineCard should render examples: InlineCard 1`] = `"<div></div>"`;
```
Comment (Crash--) [inline react/__snapshots__/examples.spec.jsx.snap]: I don't think this is want we want?

Comment (GoOz) [reply react/__snapshots__/examples.spec.jsx.snap]: Indeed… but oddly I did just like I did with others… I'll investigate further.

Comment (GoOz) [reply react/__snapshots__/examples.spec.jsx.snap]: Should be better now. :)

**Pull request merged**